api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/EmailMessagePayloadFactory.java

```diff
@@ -73,4 +106,9 @@ private static String buildUnder4PaymentSummary(PaymentCycleVoucherEntitlement v
                 voucherEntitlement.getVouchersForChildrenBetweenOneAndFour(),
                 voucherEntitlement.getSingleVoucherValueInPence());
     }
+
+    private String getRegularPaymentAmountForNextCycle(PaymentCycleVoucherEntitlement voucherEntitlement) {
+        int totalVouchersForRegularPayment = voucherEntitlement.getLastVoucherEntitlementForCycle().getTotalVoucherEntitlement() * numberOfCalculationPeriods;
```

Comment:
Why is this last voucher entitlement rather than first? Isn't the last more likely to be different than the first?

Reply:
As discussed, this is what the regular payment going forward will be.